PrivateCollections/Ethiopia/MarigetaDamdaEndalaw/MD004.xml

```diff
@@ -0,0 +1,147 @@
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS1014MD"/>
+                        <idno>MD004</idno>
+                    </msIdentifier>
+                    <msContents>
+                          <msItem xml:id="ms_i1">
+                           <title type="complete" ref="LIT3034RepCh314"/>
+                            <textLang mainLang="gez"/>
+                              <incipit>በስመ: አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩አምላክ፡ ጸሎት፡ በእንተ፡ ልሳነ፡ ሰብእ፡ ልሳነ፡ ምቀኛ፡ ወተንኮለኛ፡ ናሁ፡ ለዚባዎስሙ፡ 
+                                  በዝ፡ ድብ፡ ያዕቆብ፡ እምልሳነ፡ ሰብእ፡ ወነገደ፡ ያዕቆብ፡ ምድረ፡ </incipit>
+                              <explicit><unclear>ባህረ፡ ልሳነ፡ ፍስሐ፡ ገፁ፡ ተማህፀንኩ፡ በጸሎትከ፡ ሞተ፡ አቡ፡ ወዕቀበኒ፡ ዘበልማድ፡ እምዝ፡ ቆንተንቶሙ፡ ብከ፡ እስከተሐዉሱ፡ እምባእድ፡ 
+                                  ጥቀ፡ ሠናያቲከ፡ ብዙህ፡ እመ፡ አዕፁከ፡ እዴሁ</unclear> </explicit>
+                        </msItem>
+                        </msContents>
+                    <physDesc>
+                        <objectDesc form="Leporello">
+                        <supportDesc>
+                            <support>
+                                <material key="parchment"/>
+                            </support>
+                            <extent>
+                                <measure unit="leaf" quantity="28">28</measure>
+                 
+                            </extent>
+                            <foliation> The manuscript is not foliated. The numbers used in this description refer the panels.
+                               </foliation>
+                            </supportDesc>
+                            </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <desc>The handwriting is very difficult to read; the manuscript was copied by <persName
+                                    ref="PRS14417MarigetaDamseEndalew" role="scribe"> </persName>
+                                </desc>
+                                <seg type="ink">Black, red.</seg>
+                                <seg type="rubrication"> The word ተማህፀንኩ and personal names are rubricated.
+                                     </seg>
+                            </handNote>
+                        </handDesc>
+                        
+                       <decoDesc>
+                           <decoNote type="drawing" xml:id="d1">
+                               <locus target="#1r"/>
+                               <desc>Ṭalsam.</desc>
```

Comment:
I believe there might be a better way to markup Ṭalsam, so that they also become searchable (regardless of spelling etc). Do you remember, @CarstenHoffmannMarburg ?

Reply:
I have frequently marked such drawings in the following way:
```
<decoNote xml:id="d1" type="miniature">
                                <desc>Coloured <term key="Magic">magic picture</term>.</desc>
                            </decoNote>
```
I understood, that it is a miniature, when it has more colours that just red and black. With black or red or black and red, it is a drawing.

You can use the word Ṭalsam, as well. I propose to mark it up with the keyword "Magic" and to mark the word as foreign. I do not know, whether this markup is obligatory, but certainly an improvement as it makes the decorations searchable:

```
<decoNote type="drawing" xml:id="d1">
                               <locus target="#1r"/>
                               <desc><desc><term key="Magic"><foreign xml:lang="gez">Ṭalsam<foreign></term>.</desc>
</decoNote>
```